### PR TITLE
Docker compose can talk to each other

### PR DESCRIPTION
**Pull request by graemebates**
Unfortunately I couldn't figure out a way to check if the webserver was running in docker or not. 

Maybe we don't even need these checks, and we can always assume that they are?

Things might get changed up next week when we try to distribute everything

### CODE REVIEW

**Comment by graemebates** [comment]
I also haven't put in the fake quote server either, but that should be addressed in #27

File: transaction-server/src/server.go [hunk 1]
```diff
@@ -138,6 +133,11 @@ func addHandler(w http.ResponseWriter, r *http.Request) {
 	failOnError(err, "Failed to parse the request")
 
 	logUserCommand(req.TransactionNum, "transaction-server", "ADD", req.UserID, "", "", req.Amount)
+
+	if req.Amount < 0 {
```
Comment (sdevalapurkar) [inline transaction-server/src/server.go]: question: I am a bit confused about this negative amount stuff, so like we cant just discard the request at the web server level but we can here??

**Comment by sdevalapurkar** [comment]
also, i see the CI (dockercloud) is failing for a few tests... not sure if those need to be addressed prior to merge or not

**Comment by graemebates** [comment]
whoops this needs to go into localdev not master

File: transaction-server/src/server.go (hunk 1 above)
Comment (graemebates) [reply transaction-server/src/server.go]: yeah they need to be logged before we throw them out

Comment (sdevalapurkar) [reply transaction-server/src/server.go]: makes sense! thanks for clarification!

**Comment by sdevalapurkar** [comment]
I changed the base branch

**Comment by graemebates** [comment]
thanks!

**Comment by LeeZeitz** [comment]
Looks good, tested and working.

**Pull request merged**